src/Controller/Crudit/TemplateController.php

```diff
@@ -45,4 +49,38 @@ public function duplicate(Template $template): Response
 
         return $this->redirectToRoute('lle_hermes_crudit_template_index');
     }
+
+
+    #[Route('/copy-for-tenant/{id}', name:'lle_hermes_crudit_template_copyfortenant', methods:['GET'])]
+    public function copyForTenant(Template $template, Request $request, ParameterBagInterface $parameterBag): Response
+    {
+        /** @var class-string $tenantClass */
+        $tenantClass = $parameterBag->get('lle_hermes.tenant_class');
+        /** @var MultiTenantInterface $user */
+        $user = $this->getUser();
+        $entity = $this->em->getRepository($tenantClass)->findOneBy(['id' => $user->getTenantId()]);
+        if (!$entity || !method_exists($entity, 'getId')) {
+            $this->addFlash('danger', 'flash.no_entity_found');
```

Comment:
Crudit offers you consts for this : https://github.com/2lenet/CruditBundle/blob/e227d88ac56bc12078e89867d380e29bac1b1626/src/Brick/BrickResponse/FlashBrickResponse.php#L13